macro/src/reflect.rs

```diff
@@ -63,11 +63,17 @@ impl JavaClass {
     fn parse_javap(&self, package_name: &JavaPath) -> Result<SpannedClassInfo, SpanError> {
         let mut command = Command::new("javap");
 
+        let classpath = match env::var("CLASSPATH") {
+            Ok(val) => val,
+            Err(e) => panic!("duchess cannot read the CLASSPATH environment variable: {e}"),
+        };
+
         command
+            .arg("-cp")
+            .arg(classpath)
```

Comment:
This makes sense to me, but doesn't javap get the environment variables from the parent process anyway?

Reply:
Oh right, I forgot, it doesn't seem to